<launch>
  <rosparam command="load" file="pomdp_params.yaml" /> 
  <node name="planning_manager" pkg="minmax_pomdp" type="planning_manager.py" output="screen" >
  </node>
  <node name="game_manager" pkg="minmax_pomdp" type="game_manager.py" output="screen" >
  </node>
  <node name="decision_module" pkg="minmax_pomdp" type="decision_module.py" output="screen" >
  </node>
  <node name="game_interface" pkg="minmax_pomdp" type="game_interface.py" output="screen" >
  </node>
  <node name="robot_manager" pkg="minmax_pomdp" type="robot_manager.py" output="screen" >
  </node>

</launch>
